chore(docs): remove references to npm packages (#3676)
AztecProtocol/aztec-packages: docs/docs/dev_docs/cli/cli-commands.md

title: CLI Commands
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

Here you will find a reference to the commands available in the Aztec CLI.

## Installation

### Docker

The CLI will be installed automatically via Docker by running the command to install and start the sandbox, [instructions here](./sandbox-reference.md#with-docker).

:::info

> The `@aztec/aztec-sandbox` and `@aztec/cli` packages published to npm are deprecated in favor of Docker. If you've installed the sandbox or the CLI via NPM, **uninstall** them and remove them from your project dependencies and [install via Docker](./sandbox-reference.md#with-docker).

Author: ```suggestion
The `@aztec/aztec-sandbox` and `@aztec/cli` packages published to npm **should not be used**, in favor of Docker. If you've installed the sandbox or the CLI via NPM, **uninstall** them and remove them from your project dependencies and [install via Docker](./sandbox-reference.md#with-docker).
```
